src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -686,7 +679,11 @@ void GameObject::Update(uint32 diff)
                 }
 
                 if (target)
+                {
+                    if (target->IsInCombat() && GetMap()->GetGameTimeMS() < m_cooldownTime)
```

Comment:
Small note about those kind of changes:
I usually try to make a "sun" comment when I change something from a TC system, to help with maintenance.